run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 echo "Running Locust benchmark"
-USER_WAIT_TIME_MIN_SECONDS=0 USER_WAIT_TIME_MAX_SECONDS=0 REQUESTS_JSON="${1}" HOST="${2:-http://localhost:5000}" pipenv run locust --no-web -c 1 -r 1 -t ${3:-1m} --csv=${4:-output} -L WARNING
+USER_WAIT_TIME_MIN_SECONDS=0 USER_WAIT_TIME_MAX_SECONDS=0 REQUESTS_JSON="${1}" HOST="${2:-http://localhost:5000}" pipenv run locust --headless -u 1 -r 1 -t ${3:-1m} --csv=${4:-output} -L WARNING
```

Comment:
These changes will need to be tested in GCP as the Helm scripts will need updating - https://github.com/ONSdigital/eq-survey-runner-benchmark/blob/master/k8s/helm/values.yaml#L14